src/ruvsarpur.py

```diff
@@ -937,6 +945,19 @@ def runMain():
     deinit() #Deinitialize the colorama library
     
 
+def downloadSubtitles(subtitles, video_filename):
```

Comment:
This function needs a try/except block as we would likely not want to fail the episode downloading if something happens to the subtitles?